engines/ultima/nuvie/sound/midiparser_m.cpp

```diff
@@ -164,7 +164,7 @@ bool MidiParser_M::processEvent(const EventInfo& info, bool fireEvents) {
 }
 
 void MidiParser_M::parseNextEvent(EventInfo &info) {
-	assert(_position._playPos - _tracks[0] < _trackLength);
+	assert((uint32)(_position._playPos - _tracks[0]) < _trackLength);
```

Comment:
@NMIError is this subtraction always positive?

Reply:
Fixed in 674bd26d2613af44cfad789e081bb543656c53f0